types/__test__.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// A test file to verify types described by our .d.ts files.
+// The code in this file is only compiled, we don't run it via Mocha.
+
+import {
+  DataSource,
+  KeyValueModel,
+  ModelBase,
+  ModelBaseClass,
+  PersistedModel,
+  PersistedModelClass,
+} from '..';
+
+const db = new DataSource('db', {connector: 'memory'});
+
+//-------
+// ModelBase should provide ObserverMixin APIs as static methods
+//-------
+//
+(function() {
+  const Data = db.createModel('Data');
+
+  // An operation hook can be installed
+  Data.observe('before save', async ctx => {});
+
+  // ModelBaseClass can be assigned to `typeof ModelBase`
+  const modelTypeof: typeof ModelBase = Data;
```

Comment:
Please add a comment that `ModelBaseClass` is an alias of `typeof ModelBase` and we are testing the alias here. thx.

Reply:
See 6c38347, is my comment good enough?

Reply:
The comment is great. thx. But a tiny typo: `verifying that the value retunred `